src/mdbExtensionController.ts

```diff
@@ -366,15 +368,16 @@ export default class MDBExtensionController implements vscode.Disposable {
       EXTENSION_COMMANDS.MDB_REFRESH_DATABASE,
       (databaseTreeItem: DatabaseTreeItem): Promise<boolean> => {
         databaseTreeItem.resetCache();
+        this._explorerController.refresh();
 
-        return this._explorerController.refresh();
+        return Promise.resolve(true);
```

Comment:
There are a few of these registered commands that now return Promise.resolve(true) when they actually became sync. Is that just to not change the method signature for backwards compatibility? As far as I can tell some of the commands are sync and some are async so there seems to be precedent for making them sync if async is not necessary.

Reply:
The second argument for `this.registerCommand` should be a callback that returns a `Promise<boolean>`. For cases with async/await we can simply return true/false, but some of the functions are sync, so we force them being async by returning `Promise.resolve(true)`.

Reply:
Oh I see that here now: https://github.com/mongodb-js/vscode/blob/268cebe63758ae1f686d049786dec3857111a658/src/mdbExtensionController.ts#L215-L218

I first went looking for counter-examples in usage which is what confused me. I suppose for all those it happens to be implied.